test/powershell/engine/Module/TestModuleManifest.Tests.ps1

```diff
@@ -217,3 +217,31 @@ Describe "Tests for circular references in required modules" -tags "CI" {
         { TestImportModule $false $false $true } | ShouldBeErrorId "Modules_InvalidManifest,Microsoft.PowerShell.Commands.ImportModuleCommand"
     }
 }
+
+Describe "Test-ModuleManifest Performance bug followup" -tags "CI" {
+    BeforeAll {
+        $TestModulesFolder= 'testmodulerunspace'
+        $TestModulesPath = Join-path "$PSScriptRoot\assets" $TestModulesFolder
```

Comment:
This can be 
```Powershell
$TestModulesPath = Join-path -Path "$PSScriptRoot\assets" -ChildPath $TestModulesFolder -AdditionalChildPath 'testmodulerunspace'
```

Reply:
resolved

Reply:
resolved

Reply:
I didn't get it. Any good besides it putting two lines into one? It also throws error once I copied it. Can we just skip this comment?